frontend/packages/dev-console/src/components/topology/filters/TopologyFilterBar.tsx

```diff
@@ -17,35 +19,67 @@ type DispatchProps = {
   onFiltersChange: (filters: TopologyFilters) => void;
 };
 
+type OwnProps = {
+  visualization: Visualization;
+};
+
 type MergeProps = {
   onDisplayFiltersChange: (display: DisplayFilters) => void;
   onSearchQueryChange: (searchQuery: string) => void;
-} & StateProps;
+} & StateProps &
+  OwnProps;
 
 type TopologyFilterBarProps = MergeProps;
 
 const TopologyFilterBar: React.FC<TopologyFilterBarProps> = ({
   filters: { display, searchQuery },
   onDisplayFiltersChange,
   onSearchQueryChange,
-}) => (
-  <Toolbar className="co-namespace-bar odc-topology-filter-bar">
-    <ToolbarGroup>
-      <ToolbarItem>
-        <FilterDropdown filters={display} onChange={onDisplayFiltersChange} />
-      </ToolbarItem>
-    </ToolbarGroup>
-    <ToolbarGroup className="odc-topology-filter-bar__search">
-      <ToolbarItem>
-        <TextFilter
-          label="name"
-          value={searchQuery}
-          onChange={(e) => onSearchQueryChange(e.target.value)}
-        />
-      </ToolbarItem>
-    </ToolbarGroup>
-  </Toolbar>
-);
+  visualization,
+}) => {
+  return (
+    <Toolbar className="co-namespace-bar odc-topology-filter-bar">
+      <ToolbarGroup>
+        <ToolbarItem>
+          <FilterDropdown filters={display} onChange={onDisplayFiltersChange} />
+        </ToolbarItem>
+      </ToolbarGroup>
+      <ToolbarGroup className="odc-topology-filter-bar__search">
+        <ToolbarItem>
+          <TextFilter
+            placeholder="Find by name..."
+            value={searchQuery}
+            onChange={(e) => onSearchQueryChange(e.target.value)}
+          />
+        </ToolbarItem>
+        <ToolbarItem>
+          <Popover
+            aria-label="Find by name"
+            position="left"
+            bodyContent={
+              <>
+                <div>
+                  Search results for this filter may appear outside of the visible area.{' '}
+                  <Button variant="link" onClick={() => visualization.getGraph().fit(80)} isInline>
+                    Click here
+                  </Button>{' '}
+                  to fit all the matches to the screen.
+                </div>
+                To view hidden matches inside collapsed groups, expand the groups.
+              </>
+            }
+          >
+            <Button
+              variant="link"
+              className="odc-topology-filter-bar__info-icon"
+              icon={<InfoCircleIcon />}
```

Comment:
Do not use `icon`. Set the icon as the contents of the `Button`. Using `icon` attribute wraps the icon dom node with right margin.

Set `isInline` prop to remove the padding of the button.

Reply:
removed ```icon``` and prop and added ```isInline```.